.github/workflows/publish-realm-web.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish Realm Web
+
+on:
+  create:
+    tags:
+      - realm-web-v*
```

Comment:
Our CI for realm-js currently picks up _any_ tag & build for release, right? Or am I jumping to conclusions here?
If not, we should probably streamline the naming of these tags.